subprojects/language-java/src/test/groovy/org/gradle/api/internal/tasks/compile/DefaultJavaCompilerFactoryTest.groovy

```diff
@@ -30,7 +30,7 @@ class DefaultJavaCompilerFactoryTest extends Specification {
     Factory<JavaCompiler> javaCompilerFinder = Mock()
```

Comment:
I have a local commit that removes that no longer needed field. Ran sanityCheck and the test to validate it.